src/telemetryReporter.ts

```diff
@@ -4,7 +4,7 @@
 
 'use strict';
 
-process.env['APPLICATION_INSIGHTS_NO_DIAGNOSTIC_CHANNEL'] = true;
+ // process.env['APPLICATION_INSIGHTS_NO_DIAGNOSTIC_CHANNEL'] = true;
```

Comment:
Not sure why this is commented out? Perhaps because it was failing tslint as it should be setting a string value rather than a boolean
```suggestion
 // process.env['APPLICATION_INSIGHTS_NO_DIAGNOSTIC_CHANNEL'] = 'true';
```

Reply:
Yeah it was commented out because it was failing tslint, I'll set it as a string value. Thanks.